src/CZ.Azure.FileExchange/Client/snowpack.config.js

```diff
@@ -0,0 +1,41 @@
+// Snowpack Configuration File
+// See all supported options: https://www.snowpack.dev/#configuration
+
+/** @type {import("snowpack").SnowpackUserConfig } */
+module.exports = {
+    mount: {
+      "scripts": { url: '/' },
+      "styles": { url: '/' }
+    },
+    plugins: [
+      [
+        '@snowpack/plugin-sass', { 
+          style: "compressed",
+          sourceMap: true, 
+        }
+      ]
+    ],
+    // installOptions: {},
+    devOptions: {
+      "port": 3000,
+      "open": "none",
+      "bundle": false,
+    },
+    buildOptions: {
+      "clean": true,
+      "out": "../wwwroot/client",
+      "metaUrlPath": "/vendor"
```

Comment:
what does this?

Reply:
we don't have a dictionary with vendor right?

Reply:
This is a configuration value for SnowPack and in our case only necessary to avoid problems with its default value.

[docu here](https://www.snowpack.dev/reference/configuration#buildoptionsmetaurlpath)